metron-platform/metron-common/src/test/java/org/apache/metron/common/stellar/StellarTest.java

```diff
@@ -285,6 +285,18 @@ public void testHappyPath() {
   }
 
   @Test
+  public void testLength(){
+    String query = "LENGTH(foo)";
+    Assert.assertEquals(5, run(query,ImmutableMap.of("foo","abcde")));
+  }
+
+  @Test
```

Comment:
If I mistakenly call `LENGTH()` with no argument, then I get an `IndexOutOfBoundsException`.  And this is not unique to the `LENGTH` function.  There are a number of other Stellar functions that do the same.

This may be confusing to a user of Stellar who is not familiar with the underlying implementation.  "IndexOutOfBounds? What the heck does that mean?"  

I wonder if it makes sense to handle this in a standard way within Stellar.  Ideally we would get something along the lines of "LENGTH: Missing first argument 'input'.  Expected 'String'".  

I bring this up only as a discussion point for future work.  This PR does not necessarily have to address this situation.